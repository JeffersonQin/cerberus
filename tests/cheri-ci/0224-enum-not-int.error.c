enum e {
  A=1.
};
