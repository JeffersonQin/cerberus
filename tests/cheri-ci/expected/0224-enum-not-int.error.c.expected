cheri-ci/0224-enum-not-int.error.c:2:5: error: constraint violation: expression is not an integer constant expression
  A=1.
    ^~ 
ยง6.7.2.2#2: 
2    The expression that defines the value of an enumeration constant shall be an integer
     constant expression that has a value representable as an int.
